src/Core/src/Core/IHybridWebView.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.Maui
+{
+	public interface IHybridWebView : IView
+	{
+		string? DefaultFile { get; set; }
```

Comment:
If the handler is not going to change `DefaultFile` and `HybridRoot`, then it can be a `{ get; }`